src/test/java/org/openmrs/contrib/qaframework/page/RegistrationSummaryPage.java

```diff
@@ -14,16 +14,48 @@
 
 public class RegistrationSummaryPage extends Page {
 
+	private static final By EDIT_DEMOGRAPHICS = By.cssSelector("#edit-patient-demographics a");
+	private static final By SHOW_CONTACT_INFO = By.id("patient-header-contactInfo");
+	private static final By EDIT_CONTACT_INFO = By.id("contact-info-inline-edit");
```

Comment:
are these page elements  links, button, fields or what?

Reply:
@kdaud they are links.

Reply:
how can one tell that they are links?